frontend/src/components/Editor.js

```diff
@@ -18,26 +20,30 @@ function determineLanguage (selectedLanguage) {
   }
 }
 
-function startingCode (language) {
-  if (language === 'Python') {
-    return '## Write down your solutions here\n'
-  } else if (language === 'Java') {
-    return '/* Write down your solutions here */\n'
-  } else if (language === 'C++') {
-    return '/* Write down your solutions here */\n'
-  }
-}
+// function startingCode (language) {
```

Comment:
Will this be needed in future iterations?

Reply:
Im not sure at the moment I left it in, in case Dom wants to do anything with it